postgres/datadog_checks/postgres/statements.py

```diff
@@ -234,13 +236,32 @@ def _load_pg_stat_statements(self):
                 isinstance(e, psycopg2.errors.ObjectNotInPrerequisiteState)
             ) and 'pg_stat_statements must be loaded' in str(e.pgerror):
                 error_tag = "error:database-{}-pg_stat_statements_not_loaded".format(type(e).__name__)
-                self._log.warning(
-                    "Unable to collect statement metrics because pg_stat_statements shared library is not loaded"
+                self._check.warning(
+                    "Unable to collect statement metrics because pg_stat_statements "
+                    "extension is not loaded in database '%s'. "
+                    "See https://docs.datadoghq.com/database_monitoring/setup_postgres/"
+                    "troubleshooting#%s for more details\n%s",
+                    self._config.dbname,
+                    DatabaseConfigurationError.pg_stat_statement_not_loaded.value,
+                    warning_tags(
+                        host=self._check.resolved_hostname,
+                        dbname=self._config.dbname,
+                        code=DatabaseConfigurationError.pg_stat_statement_not_loaded.value,
+                    ),
                 )
             elif isinstance(e, psycopg2.errors.UndefinedTable) and 'pg_stat_statements' in str(e.pgerror):
                 error_tag = "error:database-{}-pg_stat_statements_not_created".format(type(e).__name__)
-                self._log.warning(
-                    "Unable to collect statement metrics because pg_stat_statements is not created in this database"
+                self._check.warning(
+                    "Unable to collect statement metrics because pg_stat_statements is not created "
+                    "in database '%s'. See https://docs.datadoghq.com/database_monitoring/setup_postgres/"
+                    "troubleshooting#%s for more details\n%s",
+                    self._config.dbname,
+                    DatabaseConfigurationError.pg_stat_statement_not_created.value,
+                    warning_tags(
+                        host=self._check.resolved_hostname,
+                        dbname=self._config.dbname,
+                        code=DatabaseConfigurationError.pg_stat_statement_not_created.value,
+                    ),
                 )
             else:
                 self._log.warning("Unable to collect statement metrics because of an error running queries: %s", e)
```

Comment:
I'm actually not sure about this one because it's not a case that has a clear root cause/solution so it doesn't fit the pattern we're trying to address here: report database configuration issues with their solution so users can act and fix themselves. 

Maybe we could have a less precise error code with a message that doesn't include troubleshooting entry URL. Personally, I'd be tempted to leave this one out until we find more common error cases that we can reliably detect and provide solutions for.